src/someipy/_internal/utils.py

```diff
@@ -43,6 +43,7 @@ def create_udp_socket(ip_address: str, port: int) -> socket.socket:
     """
     sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, socket.IPPROTO_UDP)
     sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+    sock.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
```

Comment:
For the create_udp_socket function this should not be required, as far as I can see. It can be removed here.